docker/kites/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
+MAINTAINER Simone Zaccariello <simone.zaccariello@gmail.com>
+# install binary and remove cache
+RUN apt-get update \
+    && apt-get install -y iperf3 netsniff-ng nano tcpdump screen \
```

Comment:
Multiple issues:

1) split the package list on multiple lines
2) Why install nano and screen?
3) Why you need netsniff-ng and tcpdump?

Reply:
1. Ok 👍 
2/3. I use netsniff-ng to test UDP and nano to create locally the configuration files of the UDP packets to be tested on the machine.
Screen I installed it at the request of the professor, i believe that we will use it soon, while we used tcpdump to analyze the traffic inside the pods.

Reply:
we shouldn't be using screen, nano, or tcpdump inside a container. We should gather data and analyze it outside this container (maybe in another container, but not this one).

If you need a configuration file, provide it via a ConfigMap

Reply:
@ferrarimarco we currently need to work inside a container in an interactive way... because we are in our research phase, later on we will understand how to make things automatic and maybe we can create a different container that will be used for the test environment 
we can call our container kites-test-develop to make that clear... but this is what we need right now

Reply:
> I might be missing something here. Can you please explain how is your current workflow like?

At this moment, I have to do communication tests via TCP and UDP between Nodes and Pods or between Pods and Pods using netsniff-ng and iperf3.

Reply:
we run the vagrant provision with a given cni, then we deploy the net-test_ds.yaml damonset, then we exec shell in the net-test containers and start manually running the experiments, because we need to understand:
if the connectivity is OK, it the tools works as we hope, which experiments can be performed

in a couple of weeks or so we may be able to understand which set of measurement we can run in an automated way

Reply:
Ok. Gotcha.

At the moment, which experiments/tests/connectivity checks are you running?

Reply:
they're described here
https://docs.google.com/document/d/1PIgGRPOhttMnefnZwscTuZYGbaIm21gX3Zr-m6mK-04/edit#
tcp trhoughput experiments with iperf3 
udp throughput experiments with trafgen/netsniff-ng
both for pod2pod same node, pod2pod different node, node2pod same node, node2pod different node

plus connectivity tests related to the use of services instead of ip addresses

Reply:
Ok. To avoid making this PR too big, can you/ @simonezacca open an issue to point out the fact that we need to streamline these tests?

Maybe we can update #72